index.html

```diff
@@ -19,6 +19,15 @@ <h1>Andrea Hohnsen</h1>
   </figure>
   <section id="contact">
     <h2>Contact</h2>
+    <ul>
+      <li><b>Phone:</b> +49 176 97326505</li>
+      <li><b>Email:</b><a href="mailto:a.hohnsen@mail.de">a.hohnsen@mail.de</a></li>
+      <li><b>Follow me on:</b><br><br>
```

Comment:
all fine is my opinion 👍 (in the end i think that you don' t need these 2x `br` at this place because of CSS magic)